# Genetic and antigenic characterization of an expanding H3 influenza A virus clade in US swine visualized by Nextstrain 
Figures and analysis files associated with the following manuscript: 

Megan N. Neveau, Michael A. Zeller, Bryan S. Kaplan, Carine K. Souza, Phillip C. Gauger, Amy L. Vincent, Tavis K. Anderson (unpublished). Genetic and antigenic characterization of an expanding H3 influenza A virus clade in US swine visualized by Nextstrain. 

## Abstract
Defining factors that influence spatial and temporal patterns in the diversity of influenza A virus in swine (IAV-S) is essential to inform vaccine strain selection and strategies to reduce the spread of potentially zoonotic swine-origin IAV. The relative frequency of detection of the H3 phylogenetic clade C-IVA in US swine declined to 7% in 2017, but increased to 32% in 2019. We conducted phylogenetic and phenotypic analyses of representative strains to determine putative mechanisms associated with increased detection. We created an IAV-S implementation of Nextstrain to visualize the emergence, spatial spread, and genetic evolution of H3 IAV-S. These analyses identified two C-IVA clades that emerged in 2017 and cocirculated in multiple US states. Phylodynamic analysis of the HA gene documented low relative genetic diversity from 2017 to 2019, suggesting clonal expansion. The major H3 clade was associated with an N156H amino acid substitution, although HI assays demonstrated no significant antigenic drift. The minor clade was paired with N2-02B.2 clade, acquired an N2-02A.2 in 2016, and we observed antigenic drift between the N2-02B.2 and -02A.2 clades. The major H3 clade paired with nucleoprotein (NP) of the H1N1pdm09 lineage rather than the North American swine lineage NP. Rather than genetic or antigenic diversity of the HA, our data suggest that antigenic diversity of the NA and acquisition of the H1N1pdm09 NP likely contributed to the emergence and transmission of new phylogenetic clades of C-IVA swine H3N2. 
